src/TreeTransform.jl

```diff
@@ -1,5 +1,342 @@
 module TreeTransform
 
-# Write your package code here.
+using StaticArrays
+using Rematch2: topological_sort
+import Rematch2: fieldnames
```

Comment:
I would `import Rematch2` and `using Rematch2: fieldnames`. This is the coding style in raicode and avoids accidental overloading.

Reply:
I don't need to `import` either.  I will place them both on the `using` line.